colormaps/tests.py

```diff
@@ -141,6 +142,35 @@ def test_gradient_no_labels(self):
         colormap = gradient()
         self.assertEqual(colormap.label([5, 6]), [5, 6])
 
+    def test_gradient_legend_data(self):
```

Comment:
There was a 100% test coverage. I love it. I added tests for the gradient case. Now I invite you to handle the discrete case.